src/integration/pages/idxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,100 @@
+import { runCommonCrossPlatformTests } from '../../common';
+
+export default () => {
+  runCommonCrossPlatformTests();
+
+  it('I can see the headline', () => {
+    const h1El = document.querySelector('h1');
+
+    expect(h1El).toBeInTheDocument();
+    expect(h1El.textContent).toBeTruthy();
+    expect(h1El.textContent).toMatchSnapshot();
+  });
+
+  it('I can see sections', () => {
+    const sections = document.querySelectorAll('section');
+
+    if (sections) {
+      sections.forEach(section => {
+        expect(section).toBeInTheDocument();
+
+        const h2El = section.querySelector('h2');
+
+        expect(h2El).toBeInTheDocument();
+        expect(h2El.textContent).toBeTruthy();
+        expect(h2El.textContent).toMatchSnapshot();
+      });
+    }
+  });
+
+  it('I can see Index Alsos', () => {
+    const topStories = document.querySelector(
+      '[aria-labelledby="Top-stories"]',
+    );
+
+    if (topStories) {
+      const indexAlsos = topStories.querySelector('[data-e2e=index-alsos]');
```

Comment:
Is the topstories query required? e.g. you could just query the DOM for the index-also test ID

```suggestion
      const indexAlsos = document.querySelector('[data-e2e=index-alsos]');
      
      if (indexAlsos) {
```